pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -104497,7 +104497,6 @@ self: {
        description = "A static website compiler library";
        license = stdenv.lib.licenses.bsd3;
        hydraPlatforms = stdenv.lib.platforms.none;
```

Comment:
```suggestion
```

This line should also be removed, otherwise hydra won't build this package.

Reply:
Right, I removed the line.